kubespawner/spawner.py

```diff
@@ -2677,8 +2656,9 @@ async def _make_delete_pvc_request(self, pvc_name, request_timeout):
             else:
                 raise
 
-    @_await_async_init
+    @_await_pod_reflector
     async def stop(self, now=False):
+
         delete_options = client.V1DeleteOptions()
```

Comment:
Off topic to the PR, but noted right now - this isn't a shared client but an entirely new.

Reply:
This isn't a client at all - it's the kubernetes.client package from which we import models.